aspnet/fundamentals/owin.rst

```diff
@@ -63,15 +63,15 @@ Using ASP.NET Hosting on an OWIN-based server
 OWIN-based servers can host ASP.NET applications, since ASP.NET conforms to the OWIN specification. One such server is `Nowin <https://github.com/Bobris/Nowin>`_, a .NET OWIN web server. In the sample for this article, I've included a very simple project that references Nowin and uses it to create a simple server capable of self-hosting ASP.NET 5.
 
 .. literalinclude:: owin/sample/src/NowinSample/NowinServerFactory.cs
-	:emphasize-lines: 13,19,22,27,39
+	:emphasize-lines: 13,19,22,27,41
 	:linenos:
 	:language: c#
 
-`IServerFactory <https://github.com/aspnet/Hosting/blob/1.0.0-beta5/src/Microsoft.AspNet.Hosting.Server.Abstractions/IServerFactory.cs>`_ is an ASP.NET interface that requires an Initialize and a Start method. Initialize must return an instance of `IServerInformation <https://github.com/aspnet/Hosting/blob/1.0.0-beta5/src/Microsoft.AspNet.Hosting.Server.Abstractions/IServerInformation.cs>`_, which simply includes the server's name (but the specific implementation may provide additional functionality). In this example, the ``NowinServerInformation`` class is defined as a private class within the factory, and is returned by ``Initialize`` as required.
+`IServerFactory <https://github.com/aspnet/Hosting/blob/1.0.0-rc1/src/Microsoft.AspNet.Hosting.Server.Abstractions/IServerFactory.cs>`_ is an ASP.NET interface that requires an Initialize and a Start method. Initialize must return an instance of `IFeatureCollection <https://github.com/aspnet/HttpAbstractions/blob/1.0.0-rc1/src/Microsoft.AspNet.Http.Features/IFeatureCollection.cs>`_, which we populate with a ``INowinServerInformation`` that simply wraps includes the server's name (but the specific implementation may provide additional functionality). In this example, the ``NowinServerInformation`` class is defined as a private class within the factory, and is returned by ``Initialize`` as required.
```

Comment:
`` to method names
is an [ DELETE ASP.NET] interface that requires an ``Initialize`` and a ``Start`` method. ``Initialize`` must return an instance of `IFeatureCollection <https://github.com/aspnet/HttpAbstractions/blob/1.0.0-rc1/src/Microsoft.AspNet.Http.Features/IFeatureCollection.cs>`_, which we populate with a ``INowinServerInformation`` that [DELETE simply[DELETE wraps] includes the server's name ([Delete but] the specific implementation may provide additional functionality).